browser/src/neovim/NeovimInstance.ts

```diff
@@ -543,6 +545,21 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         return this.command(`e! ${fileName}`)
     }
 
+    /**
+     * getInitVimPath
+     */
+    public getInitVimPath() {
+        const rootFolder = Platform.isWindows()
+            ? path.join(process.env["LOCALAPPDATA"], "nvim") // tslint:disable-line no-string-literal
```

Comment:
It looks like this might be duplicated from the logic in `openInitVim`? Perhaps we could refactor this such that the existence check is in a separate method - like `doesInitVimExist`? Then, we could re-use the `getInitVimPath` in `openInitVim`, so we don't have to duplicate this logic.

Reply:
I noted that and was getting itchy to refactor as you suggested but I noted theres an interim step in the `openInitVim` where it actually uses the root folder to `mkdirp(rootfolder)` so it seemed to need this interim piece of data which I guess a common method could still provide but tweaking for that seemed like increasingly diminishing returns can still move in that direction if you'd prefer

Reply:
I'd recommend refactoring this a bit - to me, `getInitVimPath` means it will return the path, and _not_ necessarily do a check to see if it exists or not. Plus, if we refactor, we can re-use the logic - and that logic for finding the `init.vim` had a few different bugs over time, so I think we should try and centralize it:
```
    /**
     * getInitVimPath
     */
    public getInitVimPath() {
         // Use path from: https://github.com/neovim/neovim/wiki/FAQ
        const rootFolder = Platform.isWindows()
            ? path.join(process.env["LOCALAPPDATA"], "nvim") // tslint:disable-line no-string-literal
            : path.join(Platform.getUserHome(), ".config", "nvim")
        const initVimPath = path.join(rootFolder, "init.vim")
        return initVimPath
    }
```

Then, we could use this in `openInitVim`:
```
    public openInitVim(): Promise<void> {
        const loadInitVim = this._configuration.getValue("oni.loadInitVim")

        if (typeof loadInitVim === "string") {
            return this.open(loadInitVim)
        } else {
            const initVimPath = this.getInitVimPath()
            const rootFolder = path.dirname(initVimPath)
            mkdirp.sync(rootFolder)

            return this.open(initVimPath)
        }
    }
```

And have a third method that checks if it exists, so we can decide if we should show the prompt:
```
public doesInitVimExist(): boolean {
    const initVimPath = this.getInitVimPath()
    try {
            return checkIfFileExistsSync(initVimPath) ? initVimPath : null
      } catch (e) {
            return null
      }
}
```